### DISCUSSION TITLE
What is CMS?

### DISCUSSION THREAD

**Discussion by clever-web** [Web-development-s/Stripe-with-laravel#17] [Q&A] [answered]
I am not familiar with CMS. Please let me know briefly.

**Comment by prowebdev119** [comment]
I can help you with this issue.
CMS means that contents management system such as wordpress, wix, shopify, woocommerce and etc.
If you need more help, please reply to me.

**Reply by clever-web** [reply]
Thank you

**Reply by superstar1205** [reply]
SCAMMERS!!!

**Comment by superstar1205** [comment, marked as answer]
?
